asmcli/components/ca/managed-cas.sh

```diff
@@ -0,0 +1,20 @@
+configure_managed_cas() {
```

Comment:
Since this is going under the experimental subcommand, can you prefix this with `x_` to follow convention?

Reply:
Done. Please take another look.